lib/detect-port.js

```diff
@@ -8,26 +8,40 @@ module.exports = (port, callback) => {
     callback = port;
     port = null;
   }
+  port = parseInt(port) || 0;
```

Comment:
default radix is 10, no need to set https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/parseInt